alphadia/transferlearning/train.py

```diff
@@ -220,22 +204,36 @@ def __init__(
         self,
         mask_modloss: bool = False,
         device: str = "gpu",
-        settings: dict | None = None,
+        lr_patience: int = 3,
+        test_interval: int = 1,
+        train_fraction: float = 0.7,
+        validation_fraction: float = 0.2,
+        test_fraction: float = 0.1,
+        epochs: int = 51,
+        warmup_epochs: int = 5,
+        batch_size: int = 1000,
+        max_lr: float = 0.0005,
+        nce: float = 25,
+        instrument: str = "Lumos",
     ):
-        if settings is None:
-            settings = {}
         super().__init__(mask_modloss, device)
+        self.lr_patience = lr_patience
```

Comment:
comparing e.g. https://github.com/MannLabs/alphadia/pull/310/files/7fefb259e23a0b6c70b9f6e3c009abc22fcf8227..842e2b840ee21725eb33083b77194c6c9cfc14d6#diff-b276775febe3c2a10ac93951d7f0d3cceeeadc0e5d162c3ec098b1bfbb8da4adR939-R942 : maybe also get rid of these instance variables .. (unless they are modified) ..

the benefit would be that in calls like the one quoted above, you can immediately tell that a constant value is being passed..